apps/api/src/migration/1660001943350-AddJobAcceptedColumnToApplicant.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddJobAcceptedColumnToApplicant1660001943350 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE ien_applicants ADD COLUMN job_accepted INT
+      CONSTRAINT ien_applicants_ien_applicant_job_fk_job_accepted REFERENCES ien_applicant_jobs (id)
+      ON UPDATE CASCADE ON DELETE CASCADE;
+      
+      UPDATE ien_applicants ia SET job_accepted = (
+        SELECT iasa.job_id FROM ien_applicant_status_audit iasa
+        WHERE ia.id = iasa.applicant_id AND iasa.status_id = 308
```

Comment:
Seeing hard coded ids in a migration makes me uneasy. I don't know if there is a fix here, but I wanted to flag it.